GliaWidgets/CallVisualizer/ScreenSharing/Coordinator/ScreenSharingCoordinator.swift

```diff
@@ -0,0 +1,58 @@
+import UIKit
+
+extension CallVisualizer {
+    final class ScreenSharingCoordinator: FlowCoordinator {
+        typealias ViewController = UIViewController
+        var delegate: ((DelegateEvent) -> Void)?
+
+        private let theme: Theme
+        private let environment: Environment
+        private var viewModel: ScreenSharingViewModel?
+        private var viewController: ScreenSharingViewController?
+
+        // MARK: - Initialization
+
+        init(
+            theme: Theme = Theme(),
+            environment: Environment
+        ) {
+            self.environment = environment
+            self.theme = theme
+        }
+
+        // MARK: - FlowCoordinator
+
+        func start() -> ViewController {
+            return showEndScreenSharingViewController()
+        }
+
+        // MARK: - Private
+
+        private func showEndScreenSharingViewController() -> ViewController {
+            typealias Props = ScreenSharingViewController.Props
+
+            let viewController = ScreenSharingViewController()
+            viewController.modalPresentationStyle = .overFullScreen
+
+            let viewModel = ScreenSharingViewModel(
+                view: viewController,
+                style: theme.screenSharing,
+                output: viewModelOutput,
```

Comment:
For consistency I think it is worth to stick to props assignment via delegate event. If there's more optimal approach, it makes sense to implement it as separate PR.

Reply:
a6dd3c3de3c9d0a64905b3c0de1f4f5242ad53c1